src/main/resources/templates/k8s_env.html

```diff
@@ -22,8 +22,8 @@ <h1>Environment Variables</h1>
             </thead>
             <tbody>
             <tr th:each="instance : ${env}">
-                <td th:text="${instance.key}">key</td>
-                <td th:text="${instance.value}">val</td>
+                <td th:text="${instance.getKey()}">val</td>
```

Comment:
Müsste hier nicht `...getKey()">key</td>` stehen?

Reply:
changed. Thanks for the remark.